fl4health/PCA/pca.py

```diff
@@ -0,0 +1,62 @@
+from logging import INFO
+from pathlib import Path
+from typing import Tuple
+
+import numpy as np
+from flwr.common.logger import log
+from flwr.common.typing import NDArray, NDArrays
+from sklearn.decomposition import PCA
+
+
+class ClientSidePCA:
+    def __init__(self, n_components: int) -> None:
+        self._pca = PCA(n_components=n_components, random_state=42)
```

Comment:
Since we are not assuming a streaming model as the paper below does, I think it is fine to borrow Sklearn's implementation for computing PCA. Let me know if this seems to be a good idea or if it would be preferable to have our own implementation.

https://arxiv.org/pdf/1907.08059.pdf